packages/gafl-webapp-service/src/pages/summary/contact-summary/__tests__/contact-summary.spec.js

```diff
@@ -114,9 +97,9 @@ describe('The contact summary page', () => {
 
   it('redirects to the contact page if no contact details have been set', async () => {
     await postRedirectGet(ADDRESS_ENTRY.uri, goodAddress)
-    const data = await injectWithCookies('GET', CONTACT_SUMMARY.uri)
-    expect(data.statusCode).toBe(302)
-    expect(data.headers.location).toBe(CONTACT.uri)
+    const data2 = await injectWithCookies('GET', CONTACT_SUMMARY.uri)
```

Comment:
Why did you rename this variable? `data` is probably clearer, `data2` makes you wonder what happened to `data`

Reply:
done - data is a bad name yes? Maybe should refactor the entire thing to responseN.